### DISCUSSION TITLE
Integration of BambuLab Studio V 1.6.2

### DISCUSSION THREAD

**Discussion by fr3ak2402** [OrcaSlicer/OrcaSlicer#888] [Q&A]
Today BambuLab Studio V 1.6.2 Hotfix1 was officially released, will the new functions of BambuLab still be included in OrcaSlicer?

**Comment by tome9111991** [comment]
Looks the nightly is merge to BS 1.6.2 Hotfix

**Reply by fr3ak2402** [reply]
Okay, where did you find the nightly build?

**Reply by tome9111991** [reply]
https://github.com/SoftFever/OrcaSlicer/actions

Choose your build for your OS

**Reply by fr3ak2402** [reply]
Okay thanks
